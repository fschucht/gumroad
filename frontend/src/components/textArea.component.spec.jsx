import renderer from 'react-test-renderer';
import { TextArea } from './textArea.component';

describe('TextArea', () => {
  it('should render a TextArea component', () => {
    const renderedComponent = renderer.create(<TextArea value='Test' placeholder='Test' />).toJSON()

    expect(renderedComponent).toMatchSnapshot()
  })
})