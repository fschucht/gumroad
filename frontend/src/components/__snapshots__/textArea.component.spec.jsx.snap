// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea should render a TextArea component 1`] = `
<textarea
  className="textarea"
  placeholder="Test"
  value="Test"
/>
`;
